tests/e2e/taqueria-plugin-taquito-flextesa.spec.ts

```diff
@@ -14,7 +14,7 @@ const exec = utils.promisify(exec1);
 const taqueriaProjectPath = 'e2e/auto-test-taquito-flextesa-plugin';
 const contractRegex = new RegExp(/(KT1)+\w{33}?/);
 let environment: string;
-let dockerName: string = 'local';
+let dockerName: string = 'taquito-flextesa';
```

Comment:
Would `local-taquito-flextesa` be more consistent with other sandbox names (such as `local-flextesa` and `local-tezos-client`)?